process/peer/process.go

```diff
@@ -650,6 +651,7 @@ func (vs *validatorStatistics) loadPreviousShardHeadersMeta(header *block.MetaBl
 			vs.storageService,
 		)
 		if err != nil {
+			log.Debug("loadPreviousShardHeadersMeta", "error", err.Error(), "prevHash", shardData.PrevHash, "shardId", shardData.ShardID, "round", shardData.Round, "nonce", shardData.Nonce)
```

Comment:
loadPreviousShardHeadersMeta.GetShardHeader

Reply:
updated comment to: "loadPreviousShardHeadersMeta after process.GetShardHeader"